libbeat/tests/compose/compose.go

```diff
@@ -43,6 +43,12 @@ func EnsureUp(t *testing.T, services ...string) {
 // EnsureUpWithTimeout starts all the requested services (must be defined in docker-compose.yml)
 // Wait for `timeout` seconds for health
 func EnsureUpWithTimeout(t *testing.T, timeout int, services ...string) {
+	// The NO_COMPOSE env variables makes it possible to skip the starting of the environment.
+	// This is useful it the service is already running locally.
+	noCompose := os.Getenv("NO_COMPOSE")
```

Comment:
Using `ParseBool` would give a bit more flexibility in the values. https://golang.org/pkg/strconv/#ParseBool

```
	if noCompose, err := strconv.ParseBool(os.Getenv("NO_COMPOSE")); err == nil && noCompose {
		return
	}
```

Reply:
changed